### PR TITLE
ctpdev : inputs shifts

### PR DESCRIPTION
**Pull request by lietava**
L0/L1 inputs shifted to LM latency

### CODE REVIEW

**Comment by lietava** [comment]
@shahor02 : please, have a look

**Review by shahor02 — DISMISSED** [review]
@lietava I did not go into details of bitmask manipulations, but overall looks fine.
Is is ready to be merged?

**Comment by lietava** [comment]
@shahor02 wait , please, for green light from me.

**Comment by lietava** [comment]
@shahor02 : 
0. I added new timeoffset   int64_t globalInputsShift  for inputs , the customOffset[16] is to be used for classes as before
1. added inputs decoding as option do CTP RawDataDecoder
2. I will add same decoding to CTF decoder as we have discussed.
This gives us both options. ( I am on holiday next week starting from tomorrow so I will modify CTF decoder after).
Please, merge this PR if ok.

File: Detectors/CTP/reconstruction/src/RawDataDecoder.cxx [hunk 1]
```diff
@@ -286,15 +290,93 @@ int RawDataDecoder::decodeRaw(o2::framework::InputRecord& inputs, std::vector<o2
...
+  if (mDoDigits && ~mDecodeInps) {
     for (auto const& dig : digitsMap) {
       digits.push_back(dig.second);
     }
   }
   // ret = 1;
   if (mStickyError) {
     if (nwrites < mErrorMax) {
-      std::string file = "/tmp/dumpCTP" + std::to_string(nwrites) + ".bin";
+      std::string file = "dumpCTP" + std::to_string(nwrites) + ".bin";
```
Comment (shahor02) [inline Detectors/CTP/reconstruction/src/RawDataDecoder.cxx:379]: Why not making the output directory settable? See e.g. how it was done for the ITS:

https://github.com/AliceO2Group/AliceO2/blob/e9e04179907dfea3486a21092ee8f44664a51800/Detectors/ITSMFT/common/workflow/src/STFDecoderSpec.cxx#L391

https://github.com/AliceO2Group/AliceO2/blob/e9e04179907dfea3486a21092ee8f44664a51800/Detectors/ITSMFT/common/workflow/src/STFDecoderSpec.cxx#L91-L96

and https://github.com/AliceO2Group/AliceO2/blob/e9e04179907dfea3486a21092ee8f44664a51800/Detectors/ITSMFT/common/reconstruction/src/RawPixelDecoder.cxx#L521-L522

**Review by shahor02 — APPROVED** [review]
Hi @lietava 

Looks fine, but I would make the dump directory configurable (can be done in a separate commit), let me know if you want me to merge this as is.

**Comment by lietava** [comment]
HI @shahor02 , actually the dump was already merged yesterday  in separate PR(11716) - I wanted to have it in P2 next Monday.  I will add configurable dir later in separate commit - thanks for examples.
Please, merge this as it is.

**Comment by shahor02** [comment]
ok, will merge once CI is passed.

**Comment by lietava** [comment]
@shahor02 : any errors related to ctp ?

**Pull request merged**